docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,219 @@
+
+To propose an enhancement for Turf, it's usually best to [start a discussion](https://github.com/Turfjs/turf/discussions) to work through the requirements. Unless you know exactly what you need! In which case [open an issue](https://github.com/Turfjs/turf/issues) to request it.
+
+To make a code contribution, follow the steps for [how to contribute](#how-to-contribute).
+
+## Architecture
+
+- GeoJSON is the lingua franca of Turf. It should be used as the data structure for anything that can be represented as geography.
+- Most work happens in sub modules. These are found in the `packages` directory prefixed with "turf-".
+- Turf modules are small, generally containing a single exported function.
+- Many turf modules depend on other smaller Turf modules.
+
+## How to contribute
+
+The best way for a new contributor to help with Turf is to find an open issue, fix it, and then share that back to the project via a pull request (PR). A maintainer will review your PR and get it merged with the rest of the code.
+
+Here is a great article on how to fork a repo, and start contributing to Turf.
+
+If you'd like to become a core contributor, just start making contributions, and inquire.
+
+### Local Setup
+
+Once you've cloned a Turf repository, and have a terminal open and currenty in the top-level turf directory. Run the following:
+- `git checkout master` (if not already)
+- `corepack enable pnpm`
+  - enable pnpm as a package manager. Requires Node 16+.  Alternatively just `npm install -g pnpm`.
+- `pnpm install`
+  - install dependencies and build modules
+- `pnpm test`
+  - run all tests and linters
+
+You're now ready to contribute.
+
+<details>
+<summary>Structure of a Turf module</summary>
+<br>
+
+Turf modules are under the `packages` directory.  For example `packages/turf-area`.  Here's how they are structured.
+
+```
+turf-<MODULE NAME>
+│   index.ts
+│   bench.ts
+│   test.ts
+│   package.json
+│   README.md
+│   LICENSE
+│
+└───test
+    │
+    ├───in
+    │   points.geojson
+    │
+    └───out
+        points.geojson
+```
+
+* `index.ts` - This file contains the definition of the function including [TypeScript](https://www.typescriptlang.org/) types, function implementation, the embedded [JSDocs](http://usejsdoc.org) documentation, and finally the single exported function that the module provides.
+* `bench.ts` - This file uses [Benchmark](https://benchmarkjs.com/) to establish benchmark performance tests for your function.
+* `test.ts` - This file includes your [tape](https://github.com/substack/tape)
+  unit tests. Tests can either be traditional code based tests, or tests that pull in data files and compare inputs to outputs (see below).
+* `package.json` - The [node](http://nodejs.org) metadata container file.
+  Modules imported in `index.ts` should be listed here under `dependencies`,
+  and modules used in `test.ts` or `bench.ts` should be listed under
```

Comment:
I was going to comment on just these two instances, but there's a lot of usages of "module" in this file that are probably more canonically called "packages". Might be worth switching around.

Reply:
Updated to use consistent terminology.